FEM/rf_pcs.cpp

```diff
@@ -815,6 +843,124 @@ void CRFProcess::Create()
 	double* nod_values = NULL;
 	double* ele_values = NULL; // PCH
 
+    if (_pcs_type == FiniteElement::HEAT_TRANSPORT_BHE)
```

Comment:
Can you derive a class from CRFProcess, and put all BHE stuff in the new class, e.g. CRFProcessBHE?